sfcli/SmartFace.Cli/Core/Domain/DataSelector/Impl/CameraODataSelector.cs

```diff
@@ -1,9 +1,8 @@
 using SmartFace.ODataClient.Default;
-using SmartFace.ODataClient.SmartFace.Domain.DataAccess.Models.Core;
 
 namespace SmartFace.Cli.Core.Domain.DataSelector.Impl
 {
-    public class CameraODataSelector : ODataSelector<Camera>, IQueryDataSelector<Camera>
+    public class CameraODataSelector : ODataSelector<ODataClient.SmartFace.Domain.DataAccess.Models.Core.Camera>, IQueryDataSelector<ODataClient.SmartFace.Domain.DataAccess.Models.Core.Camera>
```

Comment:
maybe only a pseudochange?

Reply:
This was changed due to conflict with camera from added ManagementApi namespace... resolved. Resolved by using `using` the correct namespace